tests/pytest_tests/unit_tests/test_saved_model.py

```diff
@@ -117,8 +119,10 @@ def make_local_artifact_public(art, mocker):
         },
     )
     pub._manifest = art._manifest
-    for val in pub._manifest.entries.values():
-        val.__class__ = ArtifactManifestEntryPatch
```

Comment:
I don't think this hack is necessary at all. Shouldn't it be enough to move the mocking (lines 94-96) to before the manifest entry is created (line 134)?

Reply:
I removed it: https://github.com/wandb/wandb/pull/5545